.env.local

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_RECAPTCHA_SITE_KEY=""
```

Comment:
I don't think we need the double quote since when the value is imported to the application, it will be treat as string 🤔

Reply:
For me, Do/Don't have gives the same result. but I just put the double quote in case there's provided with bare text like `fjkldol, ef893mfkf`so that you just copy and paste it inside the double quote directly. And yes, it's a string.

Reply:
Then please remove, no need for extra.